### PR TITLE
Add schema from NPM

### PR DESCRIPTION
**Pull request by peteyy**
Solves Issue: #43

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -16,7 +16,8 @@
     "caller": "^0.0.1",
     "core-util-is": "^1.0.1",
     "debuglog": "^1.0.1",
-    "enjoi": "^1.0.0"
+    "enjoi": "^1.0.0",
+    "swagger-schema-official": "^2.0.0-96305d9"
```
Comment (tlivings) [inline package.json]: Prefer this just be `"^2.0.0-" which will pick up all patches.

**Comment by peteyy** [comment]
done :+1:

File: package.json (hunk 1 above)
Comment (peteyy) [inline package.json]: done :+1:

**Comment by tlivings** [comment]
LGTM :+1:

**Pull request merged**